common/src/main/java/io/github/axolotlclient/modules/hud/gui0/hud/item/ArrowHud.java

```diff
@@ -20,70 +20,71 @@
  * For more information, see the LICENSE file.
  */
 
-package io.github.axolotlclient.modules.hud.gui.hud.item;
-
-import java.util.List;
+package io.github.axolotlclient.modules.hud.gui0.hud.item;
 
 import io.github.axolotlclient.AxolotlClientConfig.api.options.Option;
 import io.github.axolotlclient.AxolotlClientConfig.impl.options.BooleanOption;
-import io.github.axolotlclient.modules.hud.gui.entry.TextHudEntry;
+import io.github.axolotlclient.bridge.entity.AxoPlayer;
+import io.github.axolotlclient.bridge.item.AxoItemClass;
+import io.github.axolotlclient.bridge.item.AxoItemStack;
+import io.github.axolotlclient.bridge.item.AxoItems;
+import io.github.axolotlclient.bridge.render.AxoRenderContext;
+import io.github.axolotlclient.bridge.util.AxoIdentifier;
+import io.github.axolotlclient.modules.hud.gui0.entry.TextHudEntry;
 import io.github.axolotlclient.modules.hud.util.DrawPosition;
-import io.github.axolotlclient.modules.hud.util.ItemUtil;
-import net.minecraft.client.entity.living.player.LocalClientPlayerEntity;
-import net.minecraft.item.BowItem;
-import net.minecraft.item.ItemStack;
-import net.minecraft.item.Items;
-import net.minecraft.resource.Identifier;
+import io.github.axolotlclient.util.ItemUtil;
+import java.util.List;
 
 /**
  * This implementation of Hud modules is based on KronHUD.
  * <a href="https://github.com/DarkKronicle/KronHUD">Github Link.</a>
  *
  * @license GPL-3.0
  */
-
 public class ArrowHud extends TextHudEntry {
-
-	public static final Identifier ID = new Identifier("kronhud", "arrowhud");
+	private static final AxoIdentifier ID = AxoIdentifier.of("kronhud", "arrowhud");
+	private static final AxoItemStack ITEM = AxoItemStack.of(AxoItems.ARROW);
```

Comment:
This changes the modules function for newer versions. It should still also keep track of spectral & tipped arrows even if they do not exist on 1.8.9.